### DISCUSSION TITLE
Re-imagining the 'random' feature

### DISCUSSION THREAD

**Discussion by courgi** [OnionUI/Onion#1185] [Ideas]
Current implementation is a system-wide randomizer for any game extension it can fetch which is not ideal. The other negative aspect is it pulls you out to the Main UI area after randomizing. Perhaps this behavior is a product of it being an external applet enabled in Package Manager.

If possible, I would like to suggest a more targeted and desirable usage of random where depending on what game console you are currently browsing, the randomizer would limit its choice to whithin that romset and simply highlight the random game from the gamelist.  

Keep up the great work. The Miyoo Mini Plus is now my go-to gaming device ONLY because of Onion OS

**Comment by Aemiii91** [comment]
Press Y while in the game list and select "Random game" to get one from that list only.

**Comment by schmurtzm** [comment]
A great addition imo would be to be able to run again the randomize when the proposed choice is not desired (with X key for example).
